docs/standards/generic-standards/lsp14-ownable-2-step.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: 'LSP14 - Ownable 2-Step'
+sidebar_position: 3
+---
+
+# LSP14 - Ownable 2-Step
+
+:::info Standard Document
+
+[LSP14 - Ownable 2-Step](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)
+
+:::
+
+## Introduction
+
+The **[LSP14 Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)** describes an extended version of **[EIP173](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-173.md)** that uses a 2-step process to transfer or renounce ownership of a contract, instead of instant execution.
+
+In addition, this standard defines hooks that call the **[universalReceiver(...)](../smart-contracts/lsp0-erc725-account.md#universalreceiver)** function of the current owner and new owner, if these addresses are contracts that implement LSP1. This aims to:
+
+- notify when the new owner of the contract should accept ownership.
+
+![Transfer Ownership Notification](/img/standards/lsp14/transfer-ownership-notification.jpeg)
```

Comment:
drawings and illustrations are great 👌 

I think this should not be the first thing in the Standard page. There should be a separate section about **notification hooks**.